### PR TITLE
Squad Screen

### PR DESCRIPTION
**Pull request by Jarrydcrawford**
- Separated squad from Dashboard
- Updated styles for player requests and moved it to the top
- Added new font size 'rabbit' at 14px for intermediate sizes.

### CODE REVIEW

File: src/components/PlayerList.tsx [hunk 1]
```diff
@@ -1,47 +1,118 @@
...
+    width: 1px;
+    height: 1px;
+    margin: -1px;
+    border: 0;
+    padding: 0;
+    white-space: nowrap;
+    clip-path: inset(100%);
+    clip: rect(0 0 0 0);
+    overflow: hidden;
+  }
```
Comment (Jarrydcrawford) [inline src/components/PlayerList.tsx]: Should be a utility eventually but leaving it here for now 🤷‍♂️

File: src/components/PlayerList.tsx [hunk 2]
```diff
@@ -1,50 +1,121 @@
...
+      <PlayerListContainer>
         {players.map(player => (
-          <ListItem key={player.id}>
-            <CardBadge>{player.name[0]}</CardBadge>
-            <GrowingPlayerContent>
-              {player.name}{' '}
+          <PlayerCard key={player.id}>
+            <PlayerBadge size="md">
+              {player.name[0]}{' '}
               {player.user && <span title="Player has a user">🙍‍♂️</span>}
```
Comment (harrygr) [inline src/components/PlayerList.tsx]: I added this 🙍‍♂️ emoji to indicate that the player is owned by a user. But now there is a playerbadge (we should rename this to `Avatar` or something) we might get 2 characters inside the badge. Would probably be better to move it outside the badge, or can get creative with a way to indicate user ownership

Comment (Jarrydcrawford) [reply src/components/PlayerList.tsx]: I've just moved it for now, but yeah, for now, a different coloured badge (where the user has no picture) or maybe just some sort of border styling. I'm no designer :P

File: src/components/PlayerList.tsx [hunk 3]
```diff
@@ -1,50 +1,121 @@
...
+}
+
+const sizes: Record<EditCardSizes, string> = {
+  sm: '1.7rem',
+  md: '2.5rem',
+};
+
+const PlayerListContainer = styled(Vspace)`
+  padding-right: ${spacing.whale};
+  padding-left: ${spacing.whale};
```
[the remark this answers is not in the document]
Comment (harrygr) [reply src/components/PlayerList.tsx]: I'm not sure this is resolved - it now seems to overflow too much. I guess the edge of the white card should line up with the content area. We might want to increase the margin around the content area no mobile if the overflowing avatar goes off the edge of the screen as well.
Screen Shot 2019-03-11 at 09 05 40

**Pull request merged**